lighthouse-core/scripts/gcp-collection/extract-data.sh

```diff
@@ -3,21 +3,32 @@
 set -euxo pipefail
 
 ##
-# @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+# @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
 # Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 # Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
 ##
 
+# Unarchives "lhr-data-*.tar.gz" files inside gcp-data folder, places
+# them inside gcp-data/data.
+
 DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 LH_ROOT="$DIRNAME/../../.."
 GCP_DATA="$DIRNAME/gcp-data"
-cd $GCP_DATA
-touch extract_failures.log
+OUTPUT_DIR="$GCP_DATA/data"
+mkdir -p "$OUTPUT_DIR"
+
+cd "$GCP_DATA"
+
+rm -rf extract_failures.log
 
+# Change this line if you want to extract trace data instead.
```

Comment:
change this line to what?

Reply:
`traces-*.tar.gz` or w.e. You'd know if you need to change it.

Reply:
ah so not this next line, change line 26?

if you'd already know if you need to change it and what to change it to, then what's the point of the comment? :)